### DISCUSSION TITLE
[Question]Prompt permission denied after update to 8.0

### DISCUSSION THREAD

**Discussion by Varorbc** [dotnet/dotnet-docker#5163] [Q&A]
One feature in my project is to create a folder in the current running directory, which works well in 7.0, But when I upgrade to 8.0, the prompt permission is denied? What am I supposed to do?

**Comment by MichaelSimons** [comment]
@Varorbc - Can you please provide more background information?  

What .NET image are you using - e.g. what OS/distro are you using?
What container user are you running as?
Can you provide more information about the prompt you are referring to?  Are you running the container interactively?
The more details you can provide, the more likely your questions will get answered.

**Reply by Varorbc** [reply]
@MichaelSimons I based on the [documentation](https://github.com/dotnet/sdk-container-builds/blob/main/docs%2FGettingStarted.md), my application published to the linux container, 7.0 to create a directory to work well, and 8.0 prompt permission denied.

Hint, I didn't make the extra Settings.

**Reply by MichaelSimons** [reply]
It looks like you are using the CLI/SDK functionality to create a container image.  Issues/questions related SDK are tracked in https://github.com/dotnet/sdk/issues.  This repo is specific to the .NET base images.

That being said, it appears you are affected by a change where in .NET 8.0 the default user is now a rootless user.  See https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?pivots=dotnet-8-0#containeruser for details.  You can explicitly set the user in your project via the `ContainerUser` property.

I will transfer this discussion item to the SDK repo for you.

**Reply by MichaelSimons** [reply]
> I will transfer this discussion item to the SDK repo for you.

Unfortunately I can't - it you have further questions, please close this discussion topic and open a new issue at https://github.com/dotnet/sdk/issues

**Reply by Varorbc** [reply]
Okay, thank you
